webrtc.html

```diff
@@ -4626,7 +4626,8 @@ <h2>Methods</h2>
         contains unstructured binary data.</p>
         <p>Note that an <code>RTCCertificate</code> might not directly hold
         private keying material, this might be stored in a secure module.</p>
-        <p>The <code>RTCCertificate</code> object can be stored and retrieved
+        <p>The <code>RTCCertificate</code> object, with the exception of the
+        the <a>[[\KeyingMaterial]]</a> internal slot, can be stored and retrieved
```

Comment:
This isn't necessary.  The object can be stored and retrieved, including the KeyingMaterial internal slot.

Firefox implements this.  An RTCCertificate can be stored to indexedDB and later retrieved and used.  This is exactly the same process that we use for keys in WebCrypto.  See https://w3c.github.io/webcrypto/Overview.html#concepts-key-storage

Reply:
So the keying material can be stored on storage that is persistent to the browser, but there's no way of getting it out so that it can be accessed by the Javascript client, is that right?
#1694 was complaining about using the word "persistent" for that kind of storage.

Reply:
If private keying material is held in a secure module, does this affect the ability to store and retrieve the KeyingMaterial internal slot?  Just trying to understand what the requirements are for storing and retrieving certificates. Typically the term "persistent" implies the ability to persist across reboots.

Reply:
What happens here is that the keying material is stored however the browser can.  If it is in a secure module, the browser needs to store a handle to the stored keying material so that it can be used again.  If the secure module goes away, then it won't be available until it comes back.  It's still persistent though in the sense that if the machine reboots, it's still there unless the user explicitly wipes the storage (or the module).

Javascript can never directly use these keys, it can only ask the browser to use them in certain predefined ways.  Just as the browser can't use the keys directly if there is a secure module in use, it can only ask the secure module to use them in predefined ways.  For these keys, they can only be used to make new WebRTC connections.